roles/openshift_facts/tasks/main.yml

```diff
@@ -10,12 +10,19 @@
 - set_fact:
     l_is_containerized: "{{ (l_is_atomic | bool) or (containerized | default(false) | bool) }}"
 
-- name: Ensure PyYaml is installed
-  action: "{{ ansible_pkg_mgr }} name=PyYAML state=present"
+- name: Ensure required packages are installed
+  action: "{{ ansible_pkg_mgr }} name={{ item }} state=present"
+  with_items:
+    - PyYAML
+    - yum-utils
+    - dbus-python
+    - libselinux-python
+    - libsemanage-python
+    - NetworkManager
   when: not l_is_atomic | bool
 
-- name: Ensure yum-utils is installed
-  action: "{{ ansible_pkg_mgr }} name=yum-utils state=present"
+- name: Enable NetworkManager
+  service: name=NetworkManager enabled=yes state=started
```

Comment:
Hmm - I agree, but on that point it might be good to move all package / distribution oriented tasks out of openshift_facts and into a pre-req playbook/task - this would make it easier to adapt openshift-ansible for Debian, etc...